xtask/src/workload.rs

```diff
@@ -51,38 +29,38 @@ pub fn parse(
             percentage_cold_transfer: percentage_cold_transfer.unwrap_or(0),
             additional_initial_capacity,
         }),
-        "set_balance" => Box::new(CustomWorkload {
-            reads: 0,
-            writes: 100,
-            delete: 0,
-            update: 0,
+        "set_balance" | "heavy_write" => Box::new(CustomWorkload::new(
+            0,   // reads
+            100, // writes
```

Comment:
The old constructor looked like it was avoiding these comments - this seems like a step back

Reply:
Agree, but having a constructor that checks on percentage is a step forward for the feature of creating completely custom workloads